packages/@react-spectrum/s2/src/Meter.tsx

```diff
@@ -25,8 +25,10 @@ import {useDOMRef} from '@react-spectrum/utils';
 import {useSpectrumContextProps} from './useSpectrumContextProps';
 
 interface MeterStyleProps {
-  /** The [visual style](https://spectrum.adobe.com/page/meter/#-Options) of the Meter. */
-  variant: 'informative' | 'positive' | 'notice' | 'negative',
+  /** The [visual style](https://spectrum.adobe.com/page/meter/#-Options) of the Meter.
+   * @default 'informative'
```

Comment:
hmm I thought we had discussed making this one required but I can't remember. anyone?

Reply:
I just remember that `variant` used to be required in S1 but that was changed via https://github.com/adobe/react-spectrum/pull/6282.

Reply:
ohh maybe that was it

Reply:
I have a faint recollection that we may have been worried about new variants of meter coming from Spectrum, but not sure if that really connects with this prop becoming required unless the default might change in the future? In which case making this required might guard against that being a breaking change since users would need to be explicit about what variant they are rendering?

Reply:
ehh I think it's ok to match S1 if we made the change there. I just forgot about it